api.model.js

```diff
@@ -65,7 +65,7 @@ return db.query(query, [article_id]).then((result)=>{
     if(result.rows.length === 0){
         return Promise.reject({
             status: 404,
-            msg: "this id does not exist yet"
+            msg: "this article does not have any comments"
         })
```

Comment:
if the length is 0 we dont want to be rejecting here and do actually want the empty array to make it back to the test/client